### PR TITLE
Add code samples

### PR DESCRIPTION
**Pull request by bidoubiwa**
fixes: #252

### CODE REVIEW

File: .code-samples.meilisearch.yaml [hunk 1]
```diff
@@ -551,6 +540,131 @@ getting_started_search_md: |-
...
+
+  let mut file = File::open("meteorites.json")?;
+  let meteorites: Vec<Meteorite> = serde_json::from_reader(file)?;
+
+  client.index("meteorites").add_documents(&meteorites, None).await?;
+getting_started_configure_settings: |-
+  let mut synonyms = std::collections::HashMap::new();
+  synonyms.insert(String::from("wolverine"), vec!["xmen", "logan"]);
+  synonyms.insert(String::from("logan"), vec!["wolverine"]);
+
```
Comment (alallema) [inline .code-samples.meilisearch.yaml:617]:
```suggestion
```

File: .code-samples.meilisearch.yaml [hunk 2]
```diff
@@ -551,6 +540,131 @@ getting_started_search_md: |-
...
+      "title",
+      "overview",
+      "genre"s,
+      "release_date"
+    ])
+
+  client.index("meteorites").updateSettings({
+    filterableAttributes: ["mass", "_geo"],
+    sortableAttributes: ["mass", "_geo"]
+  })
```
Comment (alallema) [inline .code-samples.meilisearch.yaml:642]:
```suggestion
  let progress: Progress = client.index("meteorites").set_settings(&settings).await.unwrap();
```

Comment (alallema) [reply .code-samples.meilisearch.yaml:642]: I made a suggestion but I'm not sure of it ...

Comment (irevoire) [reply .code-samples.meilisearch.yaml:642]: There is no `Progress` anymore in this crate.
You could use a `Task` though;
```rust
let progress: Task = client.index("meteorites").set_settings(&settings).await.unwrap();
```

Here is the documentation of the `set_settings` method: https://docs.rs/meilisearch-sdk/latest/meilisearch_sdk/indexes/struct.Index.html#method.set_settings

File: .code-samples.meilisearch.yaml (hunk 1 above)
Comment (irevoire) [reply .code-samples.meilisearch.yaml:617]: :+1: 
And from what I see the other sdk are not setting the `searchable_attributes` nor the `displayed_attributes`.

**Review by irevoire — APPROVED** [review]
:+1:

**Comment by bidoubiwa** [comment]
bors merge

**Comment by brunoocasali** [comment]
bors merge

**Pull request merged**